library/swarm

```diff
@@ -0,0 +1,4 @@
+# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
+# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
+
+latest: git://github.com/docker/swarm-library-image@c0f8204b2a1bb34233cd2144df51a4de9572440c
```

Comment:
Next release, you'd remove it from here, since it wouldn't be
"officially supported" anymore, and thus we wouldn't rebuild it, but
it would still be available on the Hub for folks who need it for
whatever reason.